web/crux-ui/src/validations/deployment.ts

```diff
@@ -1,13 +1,15 @@
 import * as yup from 'yup'
 import { nameRule } from './common'
 
+const prefixRule = yup
```

Comment:
Don't we have something similar in `utils.ts`'s `slugify()`? At least for the regexp?

Reply:
We have slugify() in common.ts atm.